tests/.nftignore

```diff
@@ -2,12 +2,22 @@
 .DS_Store
 bbsplit/*.stats.txt
 fastqc/*/*.{html,zip}
+fastqc/*_fastqc.{html,zip}
```

Comment:
why this new file to ignore, is this coming from TEMPLATE?

Reply:
It was there before, just further down